bin/linz-bde-uploader-schema-load.pl

```diff
@@ -66,9 +66,12 @@ sub help
 
 if ( ${EXTENSION_MODE} )
 {
-    `$PSQL -c 'CREATE EXTENSION IF NOT EXISTS table_version;'` || die;
```

Comment:
Thanks best option is to provide argument to die. General syntax is:

some_condition || die "message";

Reply:
Added messages with a02f768